### PR TITLE
pb: expose buildInfo in get_configuration

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: pb/get_configuration.go [hunk 1]
```diff
@@ -58,17 +58,38 @@ func (c *RemoteProvisioning) Clone() *RemoteProvisioning {
...
+		CommitDate: c.GetCommitDate(),
+		ReleaseUrl: c.GetReleaseUrl(),
+	}
+}
+
+func NewGetConfigurationResponse(info *BuildInfo) *GetConfigurationResponse {
+	v := &GetConfigurationResponse{
+		BuildInfo: info,
+	}
+	return v.Clone()
```
Comment (Danielius1922) [inline pb/get_configuration.go:78]: Why clone a locally created object? Because of the `info` pointer?

File: pb/service.swagger.json [hunk 2]
```diff
@@ -1343,49 +1388,61 @@
       "properties": {
         "currentTime": {
           "type": "string",
-          "format": "int64"
+          "format": "int64",
+          "title": "@gotags: yaml:\"-\""
```
Comment (Danielius1922) [inline pb/service.swagger.json:1392]: Can't we somehow prevent this from being generated for swagger?

Comment (jkralik) [reply pb/service.swagger.json:1392]: No. I tried but I was not success. We need to create a script for that to remove `gotags`.

File: pb/get_configuration.go (hunk 1 above)
Comment (jkralik) [reply pb/get_configuration.go]: Just to fill deprecated values. Fixed.

**Review by Danielius1922 — APPROVED** [review]

**Pull request merged**